.github/workflows/publish.yaml

```diff
@@ -118,6 +127,12 @@ jobs:
             GIT_BRANCH: ${{ github.ref_name }}
             GIT_COMMIT: ${{ github.sha }}
 
+      - name: Cosign sign tags
+        run: cosign sign --yes ${TAGS}
+        env:
+          TAGS: ${{ steps.meta.outputs.tags }}@${{ steps.build-and-push.outputs.digest }}
+          COSIGN_EXPERIMENTAL: 1
```

Comment:
Want to make sure this is not necessary?

https://blog.sigstore.dev/cosign-2-0-released/

Reply:
Do you think the publish image needs to be triggered by "only push tag"?

Reply:
Nope, I think it's fair to have images aligned to the main / extra branches.
0.x could be optional, main is mandatory.